### PR TITLE
Ruowan/pass variables

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

**Review by PhoenixHe-msft — APPROVED** [review]

**Review by zhenglaizhang — CHANGES_REQUESTED** [review]

**Review by zhenglaizhang — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -3,7 +3,6 @@
 [![Build Status](https://dev.azure.com/azure-public/adx/_apis/build/status/public.Azure.oav)](https://dev.azure.com/azure-public/adx/_build/latest?definitionId=3)
```
[the remark this answers is not in the document]
Comment (ruowan) [reply README.md]: revert back. Resolved

File: lib/testScenario/postmanCollectionRunnerClient.ts [hunk 2]
```diff
@@ -687,7 +698,7 @@ export class PostmanCollectionRunnerClient implements TestScenarioRunnerClient {
           exec: this.postmanTestScript.generateScript({
             name: "AAD auth should be successful",
             types: ["ResponseDataAssertion", "OverwriteVariables"],
-            variables: new Map<string, string>([["bearerToken", "access_token"]]),
+            variables: new Map<string, string>([["bearerToken", "/access_token"]]),
```
Comment (raych1) [inline lib/testScenario/postmanCollectionRunnerClient.ts:701]: Is this intended change?

**Comment by raych1** [comment]
Looks good for the rest.

**Review by raych1 — APPROVED** [review]

File: lib/testScenario/postmanCollectionRunnerClient.ts (hunk 2 above)
Comment (ruowan) [reply lib/testScenario/postmanCollectionRunnerClient.ts:701]: Yes. We use jsonPointer to get response value.

**Pull request merged**